vercel.json

```diff
@@ -146,7 +146,7 @@
     },
     {
       "source": "/docs/start-building/ory-cli-install-use",
-      "destination": "https://www.ory.sh/docs/guides/ory-cli-install-use",
+      "destination": "https://www.ory.sh/docs/guides/cli/installation",
```

Comment:
Same here, please create a new redirect instead. Otherwise https://www.ory.sh/docs/guides/ory-cli-install-use will not work.